### PR TITLE
chore: Migration for getBaseEndpoint

### PR DESCRIPTION
**Pull request by ren-tham**

### CODE REVIEW

File: projects/schematics/src/migrations/4_0/methods-and-properties-deprecations/data/occ-endpoints.service.migration.ts [hunk 1]
```diff
@@ -14,4 +15,10 @@ export const OCC_ENDPOINTS_SERVICE_MIGRATION: MethodPropertyDeprecation[] = [
     deprecatedNode: GET_OCC_ENDPOINT,
     comment: `// ${TODO_SPARTACUS} Method '${GET_OCC_ENDPOINT}' was removed. Please use 'buildUrl' method instead with the proper parameters.`,
   },
+  {
+    class: OCC_ENDPOINTS_SERVICE,
+    importPath: SPARTACUS_CORE,
+    deprecatedNode: GET_BASE_ENDPOINT,
+    comment: `// ${TODO_SPARTACUS} Method '${GET_BASE_ENDPOINT}' was removed. Please use 'buildUrl' method instead with the proper parameters.`,
```
Comment (LTiger14) [inline projects/schematics/src/migrations/4_0/methods-and-properties-deprecations/data/occ-endpoints.service.migration.ts]: Please use `'getBaseUrl'` instead.

**Review by LTiger14 — CHANGES_REQUESTED** [review]
You need to replace all of the occurrences of `getBaseEndpoint` with `getBaseUrl` with the proper parameters and make sure nothing is broken.

File: projects/core/src/occ/services/occ-endpoints.service.spec.ts [hunk 2]
```diff
@@ -39,7 +39,7 @@ describe('OccEndpointsService', () => {
   });
 
   it('should return base endpoint', () => {
```
Comment (LTiger14) [inline projects/core/src/occ/services/occ-endpoints.service.spec.ts]: You can delete those cases they are duplicates.

**Review by LTiger14 — CHANGES_REQUESTED** [review]

File: projects/core/src/occ/services/occ-endpoints.service.spec.ts (hunk 2 above)
Comment (ren-tham) [reply projects/core/src/occ/services/occ-endpoints.service.spec.ts]: Done

**Review by LTiger14 — APPROVED** [review]

**Pull request merged**